packages/graphql-language-service-types/src/index.ts

```diff
@@ -283,11 +293,24 @@ export type CustomValidationRule = (
 
 export type Diagnostic = DiagnosticType;
 
+export type CompletionItemBase = {
+  label: string;
+  isDeprecated?: boolean;
+};
+
 export type CompletionItem = CompletionItemType & {
   isDeprecated?: boolean;
   deprecationReason?: string;
 };
 
+export type CompletionItemForCodeMirror = {
+  label: string;
+  type?: GraphQLType;
+  documentation: string | null | undefined;
```

Comment:
please use optionals `?:` instead of `| undefined`

Reply:
Sure!